vscode/webviews/chat/ChatEnvironmentContext.ts

```diff
@@ -0,0 +1,18 @@
+import { createContext, useContext } from 'react'
+
+export enum ChatClientType {
+    Web = 'web',
+    VsCode = 'vscode',
+}
+
+interface ChatEnvironmentContextData {
+    clientType: ChatClientType
```

Comment:
```suggestion
    clientType: CodyIDE
```

nit: Could we use the existing CodyIDE type here? Just to make it easier for people to be able to search `CodyIDE` and knows what needs to be updated when adding a new client :D

Reply:
Good suggestion, just did it in the last commit, thanks